soc/xtensa/esp32/linker.ld

```diff
@@ -19,28 +19,31 @@
 
 #define RAMABLE_REGION dram0_0_seg :dram0_0_phdr
 #define ROMABLE_REGION iram0_0_seg :iram0_0_phdr
+#define FLASH_CODE_REGION irom0_0_seg :irom0_0_phdr
+#define FLASH_DATA_REGION drom0_0_seg :drom0_0_phdr
 
 PROVIDE ( __stack = 0x3ffe3f20 );
 
 PROVIDE ( esp32_rom_uart_tx_one_char = 0x40009200 );
 PROVIDE ( esp32_rom_uart_rx_one_char = 0x400092d0 );
 PROVIDE ( esp32_rom_uart_attach = 0x40008fd0 );
 PROVIDE ( esp32_rom_uart_tx_wait_idle = 0x40009278 );
-PROVIDE ( esp32_rom_intr_matrix_set = 0x4000681c );
 PROVIDE ( esp32_rom_gpio_matrix_in = 0x40009edc );
 PROVIDE ( esp32_rom_gpio_matrix_out = 0x40009f0c );
 PROVIDE ( esp32_rom_Cache_Flush = 0x40009a14 );
 PROVIDE ( esp32_rom_Cache_Read_Enable = 0x40009a84 );
 PROVIDE ( esp32_rom_ets_set_appcpu_boot_addr = 0x4000689c );
 PROVIDE ( esp32_rom_i2c_readReg = 0x40004148 );
 PROVIDE ( esp32_rom_i2c_writeReg = 0x400041a4 );
+PROVIDE ( ets_printf = 0x40007d54 );
+PROVIDE ( intr_matrix_set = 0x4000681c );
```

Comment:
Just wondering why are you renaming this? This is provided by the ROM and have the same address as before.

Reply:
@dcpleung same reason as previous comment.